games-strategy/maxr/maxr-0.2.9.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+inherit cmake-utils eutils
```

Comment:
Use the desktop eclass for `doicon` and `make_desktop_entry` now.

Reply:
done